sections/semantics-scriptings.include

```diff
@@ -494,7 +494,7 @@
        internal slot's value.
   16. If the <{script}> element has an <{script/integrity}> attribute, then let <var>integrity metadata</var> be that attribute's value.
       Otherwise, let <var>integrity metadata</var> be the empty string.
-  17. Let <var>parser metadata</var> be "<code>parser-inserted</code>" if the <{script}> element has been flagged
+  17. Let <var ignore=''>parser metadata</var> be "<code>parser-inserted</code>" if the <{script}> element has been flagged
```

Comment:
why ignore? Shouldn't the mention in the step below refer to the `var`?

Reply:
Done.